.github/workflows/publish-pypi.yaml

```
@@ -0,0 +1,38 @@
+name: release
+on:
+  workflow_dispatch: null
+  release:
+    types: [ published ]
+jobs:
+  pypi-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Update system packages
+        run: sudo apt-get update -y
+
+      - name: Install make
+        run: sudo apt-get install -y build-essential
+
+      - name: Setup Python
+        uses: actions/setup-python@v2 # pin@v2
```

```suggestion
        uses: actions/setup-python@v4 # pin@v4
```
Can we pin it to v4?